features/org.csstudio.dls.feature/feature.xml

```diff
@@ -37,14 +37,6 @@
          version="0.0.0"
          unpack="false"/>
 
-   <!-- Required, but not listed, by org.epics.pvAccessJava -->
```

Comment:
Have you tried running the built product and ensuring it can connect to PVs? As this plugin wasn't listed in the build requirements it built fine without it, but wouldn't run correctly.

Reply:
Yes, I did check for that reason.